### DISCUSSION TITLE
New engine im am making

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#59] [General (Friday Night Chattin')]
When i started to work for you engine, i feel that it was cool to do, when i was bored i liked to test things and dig down in the source

So now, I've got the motivation to make my Own modified Psych Engine !
Here the link anyone wants 
https://github.com/Theoyeah/Theoyeah-Fnf-Engine

**Comment by ErenKaya31** [comment]
cool
